pkg/experiment/metastore/compaction/scheduler/scheduler_test.go

```diff
@@ -37,6 +39,9 @@ func TestScheduler_UpdateSchedule(t *testing.T) {
 		CompletedJobs: []*raft_log.CompletedCompactionJob{{
 			State: &raft_log.CompactionJobState{Name: "3"},
 		}},
+		EvictedJobs: []*raft_log.EvictedCompactionJob{{
```

Comment:
You probably also want a `scheduler.queue.put` for this

Reply:
Added some more assertions. This test is fairly formal, to be honest – I'd like to add an integration test when I get to it.